lib/main/atom/commands/initializeConfig.ts

```diff
@@ -0,0 +1,66 @@
+import {commands} from "./registry"
+import {resolveModule} from "../../../client/clientResolver"
+import {execFile} from "child_process"
+
+commands.set("typescript:initialize-config", () => {
+  return async e => {
+    const projectDirs = atom.project.getDirectories()
+
+    if (projectDirs.length === 0) {
+      e.abortKeyBinding()
+      return
+    }
+
+    let editor
+    let currentPath
+
+    try {
+      editor = atom.workspace.getActiveTextEditor()
+
+      if (!editor) {
+        throw new Error("There is no active text editor available.")
+      }
+
+      currentPath = editor.getPath()
+
+      if (!currentPath) {
+        throw new Error("There is no active filepath available.")
+      }
+    } catch (e) {
+      console.error(e.message)
+    }
+
+    const pathToTsc = await resolveModule("typescript/bin/tsc").catch(() =>
```

Comment:
:warning: Actually, after a closer look, we *really* need to pass `basedir` here, because otherwise `resolveModule` does pretty much the same as `require.resolve`. Using `resolveServer` looks even more lucrative then.

You don't really care if `resolveServer` returns `Server` since the interface *does* return path to executable, you can just ignore version by doing something like 
```ts
const pathToTsc = (await resolveServer(currentPath, 'tsc')).serverPath
```
note the `'tsc'` argument. Also probably a good idea to rename `resolveServer`, since it can resolve other stuff besides `tsserver`. I believe I mentioned already, that this `catch` can be moved to `resolveServer` too.

Reply:
I think I finally got it!